.codecov.yml

```diff
@@ -6,6 +6,7 @@ coverage:
     - "*_test.go"
     - "./mocks/*"
     - "./app/internal/identity/yoti*"
+    - "./app/internal/telemetry"
```

Comment:
maybe in the future it will contain actual code, but for now tests aren't useful (in my opinion)